docs/installation/bootstrap.rst

```diff
@@ -123,29 +168,18 @@ Bootstrap node.
     destination fields of IP packets to correspond to the MAC address(es)),
     :ref:`IP-in-IP needs to be enabled<enable IP-in-IP>`.
 
-Provision Storage for Prometheus Services
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-After bootstrapping the cluster, the Prometheus and AlertManager services used
-to monitor the system will not be running (their respective :term:`Pods <Pod>`
-will remain in a **Pending** state), because they require persistent storage to
-be available. You can either provision these storage volumes on the bootstrap
-node, or later on other nodes joining the cluster. Templates for the required
-volumes are available in :download:`examples/prometheus-sparse.yaml
-<../../examples/prometheus-sparse.yaml>`. Note, however, these templates use
-the `sparseLoopDevice` *Volume* type, which is not suitable for production
-installations. Refer to :ref:`volume-management` for more information on how to
-provision persistent storage.
+Validate the install
+^^^^^^^^^^^^^^^^^^^^
+- Check that all :term:`Pods <Pod>` on the Bootstrap node are in the
+  ``Running`` state. Note that Prometheus and Alertmanager pods will remain in
```

Comment:
From what I've seen in @lucieleonard 's PR, user-facing values such as an object's `state` are shown as bold, not monospace:

```suggestion
  **Running** state. Note that Prometheus and Alertmanager pods will remain in
```